app/models/concerns/foreman_bootdisk/compute_resources/vmware.rb

```diff
@@ -40,7 +40,11 @@ def iso_attach(iso, vm_uuid)
           'connected' => true,
           'allow_guest_control' => true
         }
+	vm = find_vm_by_uuid(vm_uuid)
```

Comment:
Can this be nil?

Should we have a block like
```
start_it_again = false
if vm.present? && vm.ready?
  vm.stop
  vm.reload
  start_it_again = true
end
client.vm_reconfig_cdrom options
vm.start if start_it_again
```

Reply:
This cannot be nil otherwise all other definitions won't work as well like iso_upload, bootdisk_datastore, reconfig_cdrom etc.